README.md

```diff
@@ -28,259 +28,49 @@ different networks, you can read more about this in [this article](https://tezos
+* [About Serokell](#about)
 
-<a name="static-linux"></a>
-## Obtain binaries from github release
+<a name="getting-binaries"></a>
+## Getting Tezos binaries
 
-Recomended way to get these binaries is to download them from assets from github release.
-Go to the [latest release](https://github.com/serokell/tezos-packaging/releases/latest)
-and download desired assets.
-
-Some of the individual binaries contain protocol name to determine
-with which protocol binary is compatible with. If this is not the
-case, then consult release notes to check which protocols are
-supported by that binary.
+The following distributions are supported by the `tezos-packaging`:
 
 <a name="ubuntu"></a>
-## Ubuntu Launchpad PPA with `tezos-*` binaries
+### [Ubuntu](./docs/distros/ubuntu.md)
 
-If you are using Ubuntu you can use PPA in order to install `tezos-*` executables.
-E.g, in order to do install `tezos-client` or `tezos-baker` run the following commands:
-```
-sudo add-apt-repository ppa:serokell/tezos && sudo apt-get update
-sudo apt-get install tezos-client
-# dpkg-source prohibits uppercase in the packages names so the protocol
-# name is in lowercase
-sudo apt-get install tezos-baker-010-ptgranad
-```
-Once you install such packages the commands `tezos-*` will be available.
+Native Ubuntu packages that can be installed with `apt-get` from a PPA.
 
 <a name="raspberry"></a>
-### Ubuntu packages on Raspberry Pi OS
-
-If you have a Raspberry Pi running the 64bit version of the official OS, you can
-use the Lauchpad PPA to install `tezos-*` executables on it as well.
+### [Raspberry Pi OS](./docs/distros/ubuntu.md#raspberry)
 
-You can add the PPA using:
-```
-# Intall software properties commons
-sudo apt-get install software-properties-common
-# Add PPA with Tezos binaries
-sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bionic main'
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 37B8819B7D0D183812DCA9A8CE5A4D8933AE7CBB
-sudo apt-get update
-```
-
-And install packages with `apt-get`, e.g. for `tezos-client`:
-```
-sudo apt-get install tezos-client
-```
+Some Raspverry systems also support packages that can be installed using `apt-get`.
 
 <a name="fedora"></a>
-## Fedora Copr repository with `tezos-*` binaries
-
-If you are using Fedora you can use Copr in order to install `tezos-*`
-executables.
-E.g. in order to install `tezos-client` or `tezos-baker` run the following commands:
-```
-# use dnf
-sudo dnf copr enable @Serokell/Tezos
-sudo dnf install tezos-client
-sudo dnf install tezos-baker-010-PtGRANAD
+### [Fedora](./docs/distros/fedora.md)
 
-# or use yum
-sudo yum copr enable @Serokell/Tezos
-sudo yum install tezos-baker-010-PtGRANAD
-```
-Once you install such packages the commands `tezos-*` will be available.
+Native Fedora packages that can be installed using using `dnf` or `yum`.
 
 <a name="linux"></a>
-## Other Linux distros usage
+### [Static binaries and other linux distros]
```

Comment:
Uh, this without a link _and_ between square brackets is odd, why not link this to the `/releases` page?

Reply:
uh, I somehow lost the link :confused: